django/db/backends/sqlite3/schema.py

```diff
@@ -11,26 +11,20 @@
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_table = "DROP TABLE %(table)s"
-    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s)"
+    sql_create_fk = None
+    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
```

Comment:
I'm not sure exactly what the ramifications might be. Anything specific to mention besides what I wrote so far?